src/components/combobox/combobox.tsx

```diff
@@ -599,6 +599,7 @@ export class Combobox
     await this.reposition();
     const maxScrollerHeight = this.getMaxScrollerHeight();
     listContainerEl.style.maxHeight = maxScrollerHeight > 0 ? `${maxScrollerHeight}px` : "";
+    listContainerEl.style.minWidth = `${referenceEl.clientWidth}px`;
```

Comment:
do you need to check to see if `referenceEl` exists here?

Reply:
`referenceEl` exists if `listContainerEl` does, right?

Reply:
Not exactly. referenceEl will depend on the user setting the referenceElement.

Reply:
I think I'm missing something. `referenceEl` is internal, [isn't](https://github.com/Esri/calcite-components/blob/master/src/components/combobox/combobox.tsx#L424) [it](https://github.com/Esri/calcite-components/blob/master/src/components/combobox/combobox.tsx#L1175)?

Reply:
Yep. my bad! I thought this was popover :) 🧠 💨